### PR TITLE
add a '-j' switch to be able to load the loaded data as jsonb field

### PR DESCRIPTION
**Pull request by madtibo**
add a `jsonfield` column of type _jsonb_ to every table by default

### CODE REVIEW

File: sql/PostLinks_pre.sql [hunk 1]
```diff
@@ -2,7 +2,8 @@ DROP TABLE IF EXISTS PostLinks CASCADE;
 CREATE TABLE PostLinks (
    Id                int         PRIMARY KEY ,
    CreationDate      timestamp   not NUll    ,
-   PostId            int         not NULL    ,
-   RelatedPostId     int         not NULL    ,
-   LinkTypeId        int         not Null
+   PostId            int         , -- not NULL    ,
```
Comment (musically-ut) [inline sql/PostLinks_pre.sql]: Is the removal of the not-NULL constraint artifacts from the PR about the Foreign keys? Is it needed for adding JSON fields?

**Review by musically-ut — COMMENTED** [review]
Thanks! This will be useful!

This feature request is ready to be merged in. Just ping me about the `not NULL` constraints and I'll pull it in.

**Comment by madtibo** [comment]
Yes, it was definitely an artefact!
It is now corrected.

**Comment by musically-ut** [comment]
I've tested this and merged it into the branch `feat-postgresql-9.4`. I _don't think_ that PostgreSQL 9.3 (the last supported version) supports `jsonb` field-type but I haven't verified it.

In any case, as the [EOL of 9.3 is this very month](https://www.postgresql.org/support/versioning/), I will merge it in as soon as 9.3 becomes unsupported.

Thanks!

**Comment by madtibo** [comment]
You are right, jsonb is a 9.4 feature.
great that everything is functional :-)

**Comment by madtibo** [comment]
@musically-ut 9.3 is now unsupported, could we merge this PR now?

**Comment by musically-ut** [comment]
Thanks for the ping, merged! \o/

**Pull request merged**